lms/djangoapps/api_manager/courses/views.py

```diff
@@ -366,6 +368,22 @@ def _get_course_data(request, course_key, course_descriptor, depth=0):
     data['resources'].append({'uri': resource_uri})
     return data
 
+def _get_static_tab_contents(request, course, tab, wrap_xmodule_display=True):
+    """
+    Wrapper around get_static_tab_contents to cache contents for the given static tab
+    """
+
+    loc = course.id.make_usage_key(
+        tab.type,
+        tab.url_slug,
+    )
+    cache_key = u'loc.{loc}'.format(loc=loc)
+    html = cache.get(cache_key)
```

Comment:
contents of the static tab is not necessarily and html string. we can rename this to `contents`.